scripts/jira_tickets/create_compara_release_JIRA_tickets.pl

```diff
@@ -71,183 +53,50 @@ sub main {
-        'metazoa'       => 'Metazoa',
-    );
-    $division ||= '';
-    if (!$division) {
-        warn "No division given, will load the division-agnostic 'relco' JSON file";
-
-    } elsif (!$capitalized_divisions{lc $division}) {
-        $logger->error("Division '$division' not recognized", 0, 0);
-    }
-    $division = $division ? $capitalized_divisions{lc $division} : 'relco';
-
-    $tickets_json = $FindBin::Bin . '/jira_recurrent_tickets.' . lc $division . '.json'
+    # Check if the introduced/default tickets JSON file exists
+    $tickets_json = $FindBin::Bin . '/jira_recurrent_tickets.' . $division . '.json'
         if !$tickets_json;
+    die "Tickets file '$tickets_json' not found! Please, specify one using -tickets option."
+        if ( !-e $tickets_json );
 
-    if ( !-e $tickets_json ) {
-        $logger->error(
-            'Tickets file '
-                . $tickets_json
-                . ' not found! Please specify one using the -tickets option!',
-            0, 0
-        );
-    }
-
-    $config = $FindBin::Bin . '/jira.conf' if !$config;
-
-    if ( !-e $config ) {
-        $logger->error(
-            'Config file '
-                . $config
-                . ' not found! Please specify one using the -config option!',
-            0, 0
-        );
-    }
-
-    printf( "\trelco: %s\n\trelease: %i\n\tdivision: %s\n\ttickets: %s\n\tconfig: %s\n\tJIRA user: %s\n",
-        $relco, $release, $division, $tickets_json, $config, $user );
+    # Ask the user to verify the parameters that are going to be used to create
+    # the tickets
+    printf( "\trelco: %s\n\trelease: %i\n\tdivision: %s\n\ttickets: %s\n\tJIRA user: %s\n",
+        $jira_adaptor->{_relco}, $jira_adaptor->{_release}, $division, $tickets_json, $jira_adaptor->{_user}
+    );
     print "Are the above parameters correct? (y,N) : ";
     my $response = readline();
     chomp $response;
-    if ( $response ne 'y' ) {
-        $logger->error(
-            'Aborted by user. Please rerun with correct parameters.',
-            0, 0 );
-    }
-
-    if ( !$password ) {
-        print 'Please type your JIRA password:';
-
-        ReadMode('noecho');    # make password invisible on terminal
-        $password = ReadLine(0);
-        chomp $password;
-        ReadMode(0);           # restore typing visibility on terminal
-        print "\n";
-    }
+    die 'Aborted by user. Please rerun with correct parameters.' if ( $response ne 'y' );
 
-    return ( $user, $relco, $release, $password, $tickets_json, $config, $division );
+    # Create JIRA tickets
+    my $subtask_keys = $jira_adaptor->create_tickets(
+        -JSON_INPUT => $tickets_json, -DRY_MODE => $dry_run);
 }
 
 
 sub usage {
     print <<EOF;
 =head1
-create_JIRA_tickets.pl -relco <string> -password <string> -release <integer> -tickets <file> -config <file> 
+create_compara_release_JIRA_tickets.pl -relco <string> -release <integer> -division <division> -tickets <file>
```

Comment:
Ensure that when these changes get merged that you update the relco document with the new script name

Reply:
The relco document was already up to date. But I'll take that into account in the future.